python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -1394,6 +1394,26 @@ def RepeatModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 1, 2))
 
 # ==============================================================================
+class RepeatInterleaveModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([3, 1, 2, 4], torch.int, True),
+    ])
+    def forward(self, x):
+        x = torch.ops.aten.repeat_interleave(x, output_size=10)
```

Comment:
```suggestion
        x = torch.ops.aten.repeat_interleave(x, output_size=10)
        y = torch.ops.aten.repeat_interleave(x)
        return x, y
```
Can you add another call without the optional output_size here?